FrontEnd/src/components/basics/Input/Input.module.scss

```diff
@@ -7,19 +7,29 @@
   }
 }
 
-.root { // used only for specificity to override bootstrap css
+.root {
+  // used only for specificity to override bootstrap css
+  & > div {
```

Comment:
nit: consider leaving a comment for why you're doing this (doesn't seem intuitive)